ironfish/src/account/accounts.ts

```diff
@@ -428,7 +428,7 @@ export class Accounts {
       for (const [accountId, decryptedNotes] of decryptedNotesByAccountId) {
         await this.db.database.transaction(async (tx) => {
           const account = this.accounts.get(accountId)
```

Comment:
Random note for later but It's possible that on a high volume account, accounts could mismatch this.accounts by the time decryption is done here causing this crash if you remove the high volume account.